charts/tekton-apps/templates/wordpress/triggerbindings.yaml

```diff
@@ -31,9 +31,9 @@ spec:
   - name: component
```

Comment:
Shouldn't we have the same changes for trigger bindings for general components? (currently we added there possibility to define own namespace, but it won't work without it as I understand)

Pls, try to add separate application as general component in different from project namespace and check that everything works.